kqueen/auth/test_ldap.py

```diff
@@ -0,0 +1,49 @@
+from .ldap import LDAPAuth
+from kqueen.models import User
+
+import pytest
+
+
+class TestAuthMethod:
+    @pytest.fixture(autouse=True)
+    def setup(self, user):
+        self.user = user
+        self.user.username = 'admin@example.org'
+        self.user.password = ''
+        self.user.save()
+
+        self.auth_class = LDAPAuth(uri='ldap://127.0.0.1:389')
+
+    def test_raise_on_missing_uri(self):
+        with pytest.raises(Exception, msg='Parameter uri is required'):
+            LDAPAuth()
+
+    def test_login_pass(self):
+        password = 'heslo123'
+        user, error = self.auth_class.verify(self.user, password)
+
+        assert isinstance(user, User)
```

Comment:
assertion error descriptions?

Reply:
already on on L26: `assert error is None`